Docs/ProjectSpec.md

```diff
@@ -72,6 +74,20 @@ Note that target names can also be changed by adding a `name` property to a targ
 	- `none`: no settings are automatically applied
 - ⚪️ **developmentLanguage**: `String` - Sets the development language of the project. Defaults to `en`
 
+### Breakpoints
+
```

Comment:
Could you list all the possible properties and what they do, similar to the other dictionaries like `Options`

Reply:
Ok, but there is a lot of options. 🙀 hehe

Reply:
Done